roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
AE and OSE have different service script names: openshift-node/master and atomic-enterprise-node/master.